webapp/src/js/components/panoptes/genome/tracks/GenotypesChannel.js

```diff
@@ -480,6 +484,41 @@ const GenotypesControls = React.createClass({
     layoutGaps: React.PropTypes.bool,
   },
 
+  // FIXME: Ideally wanted to use <QueryString> to convert Query to String
```

Comment:
OK, I'll pursue that approach, thanks.
I ran into some hurdles on the first attempt, but expect we can overcome them; I'll keep you posted.